recipes/lmdb/all/test_package/CMakeLists.txt

```diff
@@ -0,0 +1,11 @@
+cmake_minimum_required(VERSION 3.4)
+project(test_package C)
+
+include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+conan_basic_setup()
+
+find_package(PkgConfig)
+pkg_check_modules(LMDB REQUIRED IMPORTED_TARGET GLOBAL lmdb)
```

Comment:
On mobile but I think you missed declaring the pkg_config name in package I for of the recipe

Reply:
OK. I've followed the existing examples and modified the recipe to use `LMDB` for `cmake_find_package` and `cmake_find_package_multi`, and `lmdb` for `pkg_config`. I hope this is what you meant.